README.md

```diff
@@ -4,21 +4,131 @@
 - The repository hold samples about using [Spring Cloud Azure](https://learn.microsoft.com/azure/developer/java/spring-framework/) libraries. 
 - The **main** branch is using the latest stable version of Spring Cloud Azure. If you want to find sample about specific version of Spring Cloud Azure, please switch to corresponding tag in this repository.
 
+## All Samples in This Repo
+
+### Azure Active Directory
+
+| Sample Project                                                                                                                                                                                                                                                 | Support Spring Cloud Azure 4.x     | Support Spring Cloud Azure 6.x | 
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------|--------------------------------|
+| [aad-b2c-resource-server](aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server)                                                                                                                                                         | Yes                                | No                             | 
```

Comment:
How about using emoji instead of `Yes` and `No`?

:white_check_mark:
:x:

Refs: https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md